en/peripherals/frsky_telemetry.md

```diff
@@ -35,16 +73,15 @@ You can see:
 Once you have the hardware connected, follow the instructions on
 http://ilihack.github.io/LuaPilot_Taranis_Telemetry/, Section **Taranis Setup OpenTX**.
 
-If you open the `LuaPil.lua` script with a text editor, you can edit the
-configuration. Suggested modifications:
+If you open the `LuaPil.lua` script with a text editor, you can edit the configuration. 
+Suggested modifications include:
 ```
 local BattLevelmAh = -1  -- use the battery level calculation from the vehicle
 local SayFlightMode = 0  -- there are no WAV files for the PX4 flight modes
 ```
 
 
-
-## Cable/adapter sources
+## Cable/adapter sources {#ready_made_cable}
 
 Ready-made cables (which include the required adapters) are available from:

```

Comment:
@bkueng Do we know of other suppliers of ready made cables?

Reply:
Hmm, no